docker/start-containter-with

```diff
@@ -12,5 +10,5 @@ docker run --rm -ti \
   --mount type=bind,source="$data",target=/pacta-data \
   --mount type=bind,source="$report",target=/create_interactive_report \
   --mount type=bind,source="$stresstest",target=/StressTestingModelDev \
-  2diirunner:1.3.0 "$1"
+  maurolepore/2diirunner-with-packages $1
```

Comment:
@AlexAxthelm, we now run this ackward script as a terminal command, 
1. Would you use docker-compose instead? Can you draft the yaml file?
2. I haven't yet studied the difference between `--mount` and `-v` (volumes), do you have any opinion to direct my research?